### PR TITLE
update items_post for argspec

### PR DESCRIPTION
**Pull request by ezarowny**
Clubhouse link: [[ch7896]](https://app.clubhouse.io/rollbar/story/7896/pyrollbar-args-scrub-refactor)

### CODE REVIEW

**Comment by ezarowny** [comment]
@coryvirok docs updates

**Comment by ezarowny** [comment]
@brianr as requested

**Comment by ezarowny** [comment]
Is it cool to call args/kwargs deprecated?

File: api/items_post.md [hunk 1]
```diff
@@ -115,16 +115,41 @@ The example JSON payload below describes all the required and optional params th
...
+            // (DEPRECATED) Optional: kwargs
             // Object of keyword arguments (name => value) to the method/function call
             // NOTE: as this can contain sensitive data, you may want to scrub the values
             "kwargs": {
               "force": true
+            },
+
+            // Optional: argspec
+            // List of the names of the arguments to the method/function call.
+            "argspec": [request, user],
```
Comment (brianr) [inline api/items_post.md]: I think you might be missing quotes around `request` and `user`?

File: api/items_post.md [hunk 2]
```diff
@@ -115,16 +115,41 @@ The example JSON payload below describes all the required and optional params th
...
+
+            // Optional: varargspec
+            // Name of the list containing the unnamed arguments to the method/function call.
+            // THe actual list will be found in locals.
```
Comment (brianr) [inline api/items_post.md]: Typo: THe

File: api/items_post.md [hunk 3]
```diff
@@ -115,16 +115,41 @@ The example JSON payload below describes all the required and optional params th
...
+            "varargspec": args,
```
Comment (brianr) [inline api/items_post.md]: It'd be good to provide an actual example value, instead of just `args`

**Comment by brianr** [comment]
@ezarowny Yes, I think it's fine to call args/kwargs deprecated.

File: api/items_post.md (hunk 1 above)
Comment (ezarowny) [inline api/items_post.md]: I suppose they are strings in this case.

File: api/items_post.md (hunk 3 above)
Comment (ezarowny) [inline api/items_post.md]: Ah, but those are the most common names in Python.

`*args` -> args
`**kwargs` -> kwargs

Comment (brianr) [inline api/items_post.md]: Yeah but what is the format? Is args... a list of values? A list of dictionaries with some expected format? So many questions that would be easy to clearly answer with a literal example, like the rest of this doc does :)

Comment (ezarowny) [inline api/items_post.md]: Ah, it's just the name of the list containing the unnamed arguments to the method/function call. I can try to explain that further I suppose.

**Pull request merged**